installer/tue-install-impl.bash

```diff
@@ -640,7 +658,9 @@ function tue-install-system-now
         done
 
         # shellcheck disable=SC2086
-        sudo apt-get install --assume-yes $pkgs_to_install
+        sudo apt-get install --assume-yes -qq --no-install-recommends $pkgs_to_install || sudo apt-get update  # Update and try again
+        # shellcheck disable=SC2086
+        sudo apt-get install --assume-yes -qq --no-install-recommends $pkgs_to_install || exit 1
```

Comment:
By default run update first and the install. Instead of exit, use tue-install-error.
```suggestion
        sudo apt-get update -qq 
        # shellcheck disable=SC2086
        sudo apt-get install --assume-yes -qq --no-install-recommends $pkgs_to_install || tue-install-error "An error occurred in installing system packages."
```

Reply:
Nope, this was done for a speedup. An `apt-get update` is not often required (typically happens even daily in the background on most laptops. So this will save time 90% of the time costing a little extra on the 10%.
Also running `nobleo-get install` with different options multiple times will be quicker

Reply:
Changed so update is done and tmp file is created, so no new update is done.